# The bounding box of files created with the eps commands is too small.
# This simple patch seems to fix the issue. It's probably  bug in gs.
# See bug #468336.

--- a/i0/graph.i
+++ b/i0/graph.i
@@ -331,7 +331,7 @@
   close, f;
   return name+".eps";
 }
-if (is_void(EPSGS_CMD)) EPSGS_CMD= "gs -q -dNOPAUSE -dBATCH";
+if (is_void(EPSGS_CMD)) EPSGS_CMD= "gs -q -dNOPAUSE -dSAFER -dBATCH";
 
 func pdf(name)
 /* DOCUMENT pdf, name
